tracks.yaml

```diff
@@ -1,4 +1,27 @@
 tracks:
+  kinetic:
+    actions:
+    - bloom-export-upstream :{vcs_local_uri} :{vcs_type} --tag :{release_tag} --display-uri
+      :{vcs_uri} --name :{name} --output-dir :{archive_dir_path}
+    - git-bloom-import-upstream :{archive_path} :{patches} --release-version :{version}
+      --replace
+    - git-bloom-generate -y rosrelease :{ros_distro} --source upstream -i :{release_inc}
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name ubuntu
+#    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+#      -i :{release_inc} --os-name debian --os-not-required
```

Comment:
The same comment as https://github.com/ros-gbp/cartographer-release/pull/2#discussion_r177955439 applies here.